Amazon.QLDB.Driver.Tests/Amazon.QLDB.Driver.Tests.csproj

```diff
@@ -8,8 +8,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Amazon.IonDotnet" Version="0.9.0" />
-    <PackageReference Include="Amazon.IonHashDotnet" Version="0.9.0" />
+    <PackageReference Include="Amazon.IonDotnet" Version="0.9.0-beta" />
```

Comment:
You should not have to reference this directly unless you want to use a different version that what Amazon.IonHashDotnet gives you.

Reply:
Dependency removed